code/_onclick/telekinesis.dm

```diff
@@ -144,8 +144,8 @@ var/const/tk_maxrange = 15
 		apply_focus_overlay()
 		focus.throw_at(target, 10, 1,user)
 		last_throw = world.time
-		user.changeNext_move(CLICK_CD_MELEE)
```

Comment:
The click delay was the the main nerf along with the inability to sustain a grab on items that have been picked up, with click delay it's just like normal throwing